x/xion/types/msgs.go

```diff
@@ -96,7 +96,7 @@ func (msg MsgMultiSend) ValidateBasic() error {
 		return banktypes.ErrNoOutputs
 	}
 
-	return banktypes.ValidateInputsOutputs(msg.Inputs, msg.Outputs)
+	return banktypes.ValidateInputOutputs(msg.Inputs[0], msg.Outputs) // TODO change to only use one input
```

Comment:
This was meant to model the existing Bank MultiSend, what does it do here?

Reply:
it recreates the same logic as before except the inputs is an array. the above code makes sure the length is always 1 making this check safe